packages/theme/src/theme/default.ts

```diff
@@ -122,6 +122,7 @@ export const theme = {
     space0: "0rem", // 0px
     space10: "0.125rem", // 2px
     space20: "0.25rem", // 4px
+    space25: "0.4rem", // 6px
```

Comment:
Not sure if we should add this token. Do we need to use it only in this component or across the app? Because I think the design team hasn't added it as a token. What do you think @richbachman?